src/Subsystems/Intake.h

```diff
@@ -8,8 +8,7 @@ class Intake: public frc::Subsystem {
 private:
 	static const char kSubsystemName[];
 	static std::shared_ptr<Intake> self;
-  WPI_TalonSRX leftIntake;
-  WPI_TalonSRX rightIntake;
+  WPI_TalonSRX intakeMotor;
```

Comment:
Let's see ... the name of the class is `Intake` and there is a member called `intakeMotor`. Seems a bit redundant. We still do this, but we should consider different naming where the name of the class is not part of the member variable. 

e.g. `motor`

No change necessary.